client/js/meshes/hitboxMesh.js

```diff
@@ -0,0 +1,31 @@
+define(function(require) {
+
+var THREE = require("THREE");
+
+return function HitboxMesh(halfExtents) {
+	var self = this;
+
+	var material = new THREE.MeshBasicMaterial({
+		color: 0x000000,
+		wireframe: true
+	});
+
+	var geometry = new THREE.CubeGeometry(
+			halfExtents.x * 2,
+			halfExtents.y * 2,
+			halfExtents.z * 2);
+	var mesh = new THREE.Mesh(geometry, material);
+
+	self.equals = function(other) {
+		return halfExtents.x === other.x && halfExtents.y === other.y && halfExtents.z === other.z;
```

Comment:
Poorly named... maybe? I mean, it's used to check when the hitbox mesh should change. That is, when the half extends of the parent object changes. So you pass the halfextents and it checks if it is the same. If not, you update the hitbox with a new size, which requires reconstructing all the geometry.